recipes/GKE Setup/deploy.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -e
+
+GOOGLE_PROJECT=""
+ORG_ID=""
+BIGQUERY_DATASET="fullstory-export-data"
+SERVICE_NAME="hauser-svc"
+GCS_BUCKET="$GOOGLE_PROJECT-hauser"
```

Comment:
I think the script needs to be tested in an empty/dummy project. For example,`GOOGLE_PROJECT` var is empty at this point, which means this script cannot work.

Reply:
You could remove the command-line args and make the interface entirely driven by env vars. Then you can rely on `GOOGLE_PROJECT` being set (since the script will abort if it is not). To preserve defaults for the ones below, you can do this:
```sh
BIGQUERY_DATASET="${BIGQUERY_DATASET:-fullstory-export-data}"
```